sdk/remoterendering/azure-mixedreality-remoterendering/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Release History
 
+## 1.0.0b3 (2022-07-04)
+
 ## 1.0.0b2 (Unreleased)
```

Comment:
Why do you have 1.0.0b3 here when 1.0.0b2 says it is unreleased?

Reply:
As far as I know 1.0.0b2 was released, but simply no date was given (at least I found the PR into main with the canges from that release). I left it as unreleased as I was not sure, whether I should retroactively change it to the 'correct' date or whether I should just leave it.

I can dig up the date again and change it to the actual release date.

Reply:
Timestamps should now be fixed.